### PR TITLE
Shona docu665

### PR DESCRIPTION
**Pull request by shonaisbister-disco**
Update intro paragraph with  new marketing copy: https://jira.wandisco.com/browse/DOCU-665

I also pruned this page a little and removed an out of date graphic. A new version can be added back in if necessary before release but it's not worth holding up the docs over.

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -1,34 +1,21 @@
...
 
-LiveData Migrator **migrates HDFS data** to other Hadoop-compatible file systems and cloud storage, including the ongoing changes made to those data before, throughout and after migration. Manage your migrations through a comprehensive and intuitive **command-line interface** or from the self-documenting **REST API**, track and **monitor migration progress**, and take advantage of optional management from WANdisco's **browser-based user interface** and deep integration with **cloud vendors' management interfaces**.
-
-The features of LiveData Migrator are designed to work together to provide migration capabilities for big data environments that are powerful, yet extremely easy to use. It combines:
+Manage your migrations through a **user interface**, a **command-line interface** or from the self-documenting **REST API**, track and **monitor migration progress**, and take advantage of optional management from WANdisco's **browser-based user interface** and deep integration with **cloud vendors' management interfaces**.
 
 - A robust and efficient scanner that visits source items only once to identify content for migration,
```
Comment (benradstone) [inline docs/about.md:20]: Lines 20-21, full stops over commas?

File: docs/about.md [hunk 2]
```diff
@@ -39,16 +26,14 @@ The features of LiveData Migrator are designed to work together to provide migra
...
-- **reduce the cost and risk** of bringing your business-critical data to [Azure Data Lake Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction), [AWS S3](https://aws.amazon.com/s3/), [Google Cloud Storage](https://cloud.google.com/storage), or to other [HDFS](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html) or [Hadoop-compatible storage platforms](https://cwiki.apache.org/confluence/display/HADOOP2/HCFS), and
-- **continue to run** your on-premises big data operations while migration is in progress.
-
-Take advantage of the WANdisco LiveData Platform to solve the challenges of migrating large data volumes from Hadoop to the cloud and keeping those data consistent across storage systems throughput migration, _even while data are under continual change_. It employs a unique, wide-area network capable consensus engine to achieve data consistency and to implement migration of data with consistency guarantees while data are ingested or modified.
+- Rapidly **provision a Live Data service** that supports simple and advanced migration needs
```
Comment (benradstone) [inline docs/about.md:29]: Lines 29-31, I really don't like the sentence flowing from one bullet point to the other. Can it just end with a full stop, and the sentences closed up?

**Review by benradstone — CHANGES_REQUESTED** [review]
Just a few comments regarding grammar/style.

File: docs/about.md (hunk 2 above)
Comment (shonaisbister-disco) [reply docs/about.md:29]: Good spot, I hadn't noticed that. I tidied up all the 'or's as well.

**Comment by shonaisbister-disco** [comment]
Added all peer review edits.

**Pull request merged**